insights/parsers/insights_client_tags.py

```diff
@@ -0,0 +1,52 @@
+"""
+InsightsClientTags - file ``/etc/insights-client/tags.yaml``
+============================================================
+"""
+
+from insights import parser
+from insights.core import YAMLParser
+from insights.specs import Specs
+
+
+@parser(Specs.insights_client_tags)
+class InsightsClientTags(YAMLParser):
+    """
+    This class parses the YAML-formatted file `/etc/insights-client/tags.yaml`.
+    The file is created by the user after running this command:
+
+    `insights-client --group=<group-name>`
+
+    The command above creates the file and adds an initial group tag
+    as defined by the `--group` option.
+    Other tags are added manually by the user.
+
+    Sample input data::
+
+        group: eastern-sap
+        location: Boston
+        description:
+        - RHEL8
+        - SAP
+        key 4: value
```

Comment:
I would remove this line because the example shows "real data", so having value as `key 4: value` did not correspond to `group: eastern-sap`, `location: Boston`.